docs/source/javascripts/all.js

```diff
@@ -1,24 +1,77 @@
 /* global places */
```

Comment:
You can use npm test command to get linting results. And also install sublime linter and sublime linter eslint packages for sublime text